README.md

```diff
@@ -50,7 +50,7 @@ client = QdrantClient(path="path/to/db")  # Persists changes to disk, fast proto
 
 ### Client-Server
 
-This is the recommended method for production usage. To run the container, use the command:
+To run the container, use the command:
```

Comment:
I'd prefer to have some sort of recommendation e.g. prefer client-server for production usage because we do get questions on why is ":memory:" mode not fast enough or similar

Reply:
I could make a separate "Production Deployment" section right underneath the "Client-Server" section, would that help?

Or I could leave the "recommended for production" phrase and just update this existing section to match QUICK_START.md, which mentions enabling authentication and `--read-only`.

Do you have a preference?

Reply:
I have a mild preference for a separate Production Deployment section — but both solutions work for the problem to be solved here.

Reply:
I tried a mixture of the two ideas, since it seemed like a new section would be easily overlooked. Let me know if it's better!